samples/ClassroomLabs/Modules/Library/Tests/Utils.ps1

```diff
@@ -0,0 +1,63 @@
+Import-Module $PSScriptRoot\..\Az.LabServices.psm1
+
+$rgName = 'TestPerm'
+$rgLocation = 'West Europe'
+$labName = 'FastLab'
+$laName = 'LATest'
+$imgName = 'CentOS-Based*'
+$maxUsers = 2
+$usageQuota = 30
+$usageAMode = 'Restricted'
+$shPsswd = $false
+$size = 'Medium'
+$title = 'Advancing Differentiation Workshop'
+$descr = 'Bringing it to the 21st Century'
+$userName = 'test0000'
+$password = 'Test000'
+
+function Get-FastLabAccount {
+    [CmdletBinding()]
+    param()
+
+    # Creat RG, Lab Account and lab if not existing
+    if (-not (Get-AzResourceGroup -ResourceGroupName $rgName -EA SilentlyContinue)) {
+        New-AzResourceGroup -ResourceGroupName $rgName -Location $rgLocation | Out-null
+        Write-Verbose "$rgname resource group didn't exist. Created it."
+    }
+
+    $la = Get-AzLabAccount -ResourceGroupName $rgName -LabAccountName $laName
+    if (-not $la) {
+        $la = New-AzLabAccount -ResourceGroupName $rgName -LabAccountName $laName
+        Write-Verbose "$laName lab account created."                
+    }
+    return $la
+}
+
+function Get-FastLab {
+    [CmdletBinding()]
+    param()
+
+    $la = Get-FastLabAccount
+                
+    $lab = $la | Get-AzLab -LabName $labName
+    if ($lab) {
+        return $lab
```

Comment:
Do we need to ensure the lab account is in any particular state?  I see that there was some handling for this in one of the tests, but it might be better to reset to a baseline here?  (can be done later, not required for this PR)

Reply:
Yes! I want to be able to do multiple runs in parallel accessing the same lab. The tests as written now don't allow this. What you suggest doesn't allow it either as, while the baseline is resetting, another test run might be changing the lab. It needs to be a mix of writing the test in an atomic way and judicious use of mutex when nothing else works. I have ideas on all of that, but they are worth a separate PR.